tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/Services/GoLanguageRepoServiceTests.cs

```diff
@@ -0,0 +1,138 @@
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using Azure.Sdk.Tools.Cli.Services;
+using System.Threading.Tasks; // added for async SetUp
+
+namespace Azure.Sdk.Tools.Cli.Tests.Services
+{
+    internal class GoLanguageRepoServiceTests
+    {
+        private string GoPackageDir { get; set; }
+        private string GoProgram => RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "go.exe" : "go";
+
+        private GoLanguageRepoService LangService { get; set; }
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            var paths = Environment.GetEnvironmentVariable("PATH")?.Split(Path.PathSeparator) ?? Array.Empty<string>();
```

Comment:
There must be a better way to check this, e.g. on linux/mac you can run `which`.

Reply:
Scanning ALL the paths in PATH seems like a bunch of extra time to take at test start.

Reply:
I think overall each language service should have a `EnsureDependencies` method or something that can be invoked by test instead.